pkg/migration/controllers/migrationschedule.go

```diff
@@ -408,3 +486,88 @@ func (m *MigrationScheduleController) createCRD() error {
 
 	return apiextensions.Instance().ValidateCRD(resource, validateCRDTimeout, validateCRDInterval)
 }
+
+func getMigratedAppStatus(migrationSchedule *stork_api.MigrationSchedule) (bool, error) {
+	for _, ns := range migrationSchedule.Spec.Template.Spec.Namespaces {
+		deployList, err := apps.Instance().ListDeployments(ns, meta.ListOptions{LabelSelector: fmt.Sprintf("%v=%v", StorkMigrationAnnotation, "true")})
```

Comment:
Don't we need to do this for other kinds as well? May be we can move the code from storkctl package into a common utils package that can be used for activating/de-activating the supported resources